pom.xml

```diff
@@ -1946,11 +1938,6 @@
               <artifactId>wildfly-openssl</artifactId>
               <version>1.1.3.Final</version>
             </dependency>
-            <dependency>
```

Comment:
Why removed?

Reply:
It was added explicitly, because we needed to bump this dependency. 

Now the BOM has this version as well, so it's not necessary to have it as another explicit dependency